edb/graphql/codegen.py

```diff
@@ -130,7 +130,9 @@ def visit_Directive(self, node):
         self._visit_arguments(node)
 
     def visit_StringValue(self, node):
-        self.write(f'"{node.value}"')
+        # only need to escape '"'
+        esc = node.value.replace('"', r'\"')
```

Comment:
What if a string has a trailing slash, e.g. `blah\`

Reply:
Right. This should be using the JSON converter to get the correct string representation since the underlying value is a Python string. I've updated the handling of StringValue node in codegen and translator (translator uses `from_python`).